src/commands/blacklist.js

```diff
@@ -8,23 +8,25 @@ module.exports = {
     .setDescription('returns registered blacklist')
     .addStringOption(option => option
         .setName('email')
-        .setDescription('register blacklisted emails (add multiple separated by \',\'), (pass "-" to clear)')
+        .setDescription('register blacklisted emails (add multiple separated by ","), (pass "-" to clear)')
         ),
     async execute(interaction) {
         const blacklist = interaction.options.getString('email');
         await database.getServerSettings(interaction.guildId, async serverSettings => {
             if (!blacklist) {
-                await interaction.reply("Blacklisted names: " + serverSettings.blacklist.toString());
+                await interaction.reply("Blacklisted emails:\n" + serverSettings.blacklist.join(", "));
             } else if (blacklist === "-") {
                 serverSettings.blacklist=[];
                 database.updateServerSettings(interaction.guildId, serverSettings);
                 await interaction.reply("Blacklist cleared!");
             }
             else {
-                    const blacklistedNames = serverSettings.blacklist.concat(blacklist.split(",").map(name=> name.trim()));
-                    serverSettings.blacklist = blacklistedNames;
+                    const newBlacklists = blacklist.split(",").map(name=> name.trim())
+                    const blacklistedNames = (format = false) => serverSettings.blacklist
+                        .concat(format ? newBlacklists.map(v=> `**${v}**`) : v);
```

Comment:
v should probably be newBlacklists

Reply:
Im all for renaming to your likings, but not really certain about using a plural here. `newBlacklists` is an array. Each value (currently `v`) corresponds to an individual item. Therefore `newBlacklist` should be more suitable. Let me now your thoughts 😀

Reply:
I didn't mean naming conventions. If I understand it correctly, it should be:
```
.concat(format ? newBlacklists.map(v=> `**${v}**`) : newBlacklists);
```
v isn't defined after the map scope and wouldn't really make a lot of sense.